### PR TITLE
kubelet-in-userns.md: update for minikube

### PR DESCRIPTION
**Pull request by AkihiroSuda**
minikube now supports Rootless Docker driver.

minikube internally sets the KubeletInUserNamespace feature gate automatically for supporting Rootless Docker driver.

https://minikube.sigs.k8s.io/docs/drivers/docker/

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/kubelet-in-userns.md [hunk 1]
```diff
@@ -34,14 +34,21 @@ If you are just looking for how to run a pod as a non-root user, see [SecurityCo
...
 
-<!--
-[minikube](https://minikube.sigs.k8s.io/docs/) also plans to support Rootless Docker/Podman drivers.
-See [minikube issue #10836](https://github.com/kubernetes/minikube/issues/10836) to track the progress.
--->
+### minikube
+
+[minikube](https://minikube.sigs.k8s.io/) also supports running Kubernetes inside Rootless Docker.
+
+See [minkube Documentation/Drivers/docker](https://minikube.sigs.k8s.io/docs/drivers/docker/).
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubelet-in-userns.md:47]:
```suggestion
See the page about the [docker](https://minikube.sigs.k8s.io/docs/drivers/docker/) driver
in the Minikube documentation.
```

File: content/en/docs/tasks/administer-cluster/kubelet-in-userns.md [hunk 2]
```diff
@@ -34,14 +34,21 @@ If you are just looking for how to run a pod as a non-root user, see [SecurityCo
...
+
+Rootless Podman is not supported yet.
+
+<!-- Supporting rootless podman is discussed in https://github.com/kubernetes/minikube/issues/8719 -->
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubelet-in-userns.md:51]: We [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future), so I'd either omit the “yet” or leave out the whole paragraph.

**Review by sftim — COMMENTED** [review]
Hi @AkihiroSuda 

Thanks for the PR. Here's a review with two suggestions.

**Comment by jimangel** [comment]
Looks good after addressing @sftim's feedback.
/lgtm

File: content/en/docs/tasks/administer-cluster/kubelet-in-userns.md (hunk 1 above)
Comment (AkihiroSuda) [reply content/en/docs/tasks/administer-cluster/kubelet-in-userns.md:47]: fixed

File: content/en/docs/tasks/administer-cluster/kubelet-in-userns.md (hunk 2 above)
Comment (AkihiroSuda) [reply content/en/docs/tasks/administer-cluster/kubelet-in-userns.md]: removed "yet"

**Comment by natalisucks** [comment]
Seconding Jim's LGTM

**Review by sftim — COMMENTED** [review]
/approve

Thanks @AkihiroSuda

**Comment by AkihiroSuda** [comment]
What's current status?

**Comment by sftim** [comment]
[Changes](https://github.com/kubernetes/website/compare/4c280bec673b7d58fcbcf08c8489516114741b5d..bc0a2487f8684728ab5a2639ea8d0e743fe2393b) since last LGTM are negligible.

/lgtm

**Pull request merged**